forms/company_admin_management.py

```diff
@@ -1,5 +1,4 @@
 #  Nikulin Vasily © 2021
-
```

Comment:
What is wrong with empty lines?

Reply:
It's just a code style. Some files used to have empty line before copyrite, some didn't use to. I decided to delete them